app/resources/api/v1/activity_resource.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::ActivityResource < JSONAPI::Resource
+  model_name 'Activity'
+  immutable
+
+  attributes :plan_id, :minutes_per_week
```

Comment:
```suggestion
  attributes :plan_id, :minutes_per_week, :tag_ids
```

We're going to need to include the tag_ids as well so the categorisation etc can be extracted.